util/test/start_test.py

```diff
@@ -1121,11 +1122,16 @@ def set_up_executables():
         prediff_for_slurm = os.path.join(util_dir, "test", "prediff-for-slurm")
         if prediff_for_slurm not in chpl_system_prediff:
             chpl_system_prediff.append(prediff_for_slurm)
-    elif 'lsf-' in launcher:
+    if 'lsf-' in launcher:
         # With lsf-based launcher, auto-run prediff-for-lsf.
         prediff_for_lsf = os.path.join(util_dir, "test", "prediff-for-lsf")
         if prediff_for_lsf not in chpl_system_prediff:
             chpl_system_prediff.append(prediff_for_lsf)
+    if 'ucx' in comm_substrate:
+        # Quiet warnings about UCX being experimental
```

Comment:
Is it reasonable to expect that over time we might accumulate more filtered-out messages related to UCX? If so, should we maybe just say "spurious UCX-related messages" here instead of "warnings about UCX being experimental"?  (Because someone doing a commit to update the prediff may not think to look here to update the comment.)

Reply:
I doubt it. Longer term I expect gasnet-ucx to be like gasnet-ibv in this regard and we don't have any general filters there.

Reply:
Good enough for me, then!

Reply:
Eh, I changed it to be more generic anyways